src/listeners/messageCreate.ts

```diff
@@ -7,6 +7,7 @@ import { sendKickEmbed } from '../utils/embeds';
 
 const HONEYPOT_CHANNEL_ID: string = vars.HONEYPOT_CHANNEL_ID;
 const ANNOUNCEMENTS_CHANNEL_ID: string = vars.ANNOUNCEMENTS_CHANNEL_ID;
```

Comment:
check these linting errors, typically if you installed a linter into your IDE these should be autofixed.